sdk/attestation/tests.yml

```diff
@@ -0,0 +1,36 @@
+trigger:
+  none
+
+jobs:
+ - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+   parameters:
+     ServiceDirectory: attestation
+     Location: westus
+     DeployArmTemplate: true
+     EnvVars:
+       ATTESTATION_CLIENT_ID: "$(ATTESTATION_CLIENT_ID)"
```

Comment:
Where are these variables coming from? I don't see them in the arm template nor do I see them in the variable group hooked up to the pipeline. I suspect these are dead code and if so it would be good to remove them.

Reply:
The variables come from the test-resources.json file and the test-resources-pre.ps1 script. MAA requires some fairly specific configuration before it can work :(.